bundles/org.eclipse.pde.doc.user/topics_WhatsNew.xml

```diff
@@ -6,7 +6,6 @@
      are made available under the terms of the Eclipse Public License v1.0
      which accompanies this distribution, and is available at
      http://www.eclipse.org/legal/epl-v10.html
-
```

Comment:
Please don't change unrelated parts in the PR. Please review the diff in Eclipse before updating the PR.